modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/converter/ObjectFieldDTOConverter.java

```diff
@@ -102,6 +107,28 @@ public ObjectField toDTO(
 		};
 	}
 
+	private boolean _isUnique(
+		List<com.liferay.object.model.ObjectFieldSetting> objectFieldSettings) {
+
+		if (ListUtil.isEmpty(objectFieldSettings)) {
+			return false;
+		}
+
+		for (com.liferay.object.model.ObjectFieldSetting objectFieldSetting :
+				objectFieldSettings) {
+
+			if (!StringUtil.equals(
+					objectFieldSetting.getName(), "uniqueValues")) {
```

Comment:
better to use `ObjectFieldSettingContants.NAME_UNIQUE_VALUES` from `object-api` module

Reply:
I think this could be done is a simpler way:
```java
private boolean _isUnique(ObjectField objectField) {
	if (Objects.equals(
			ObjectFieldSettingUtil.getValue(
				ObjectFieldSettingConstants.NAME_UNIQUE_VALUES,
				objectField),
			"true")) {

		return true;
	}

	return false;
}

```

What do you think?